### PR TITLE
automated 3 UI tests and fix upgrade typo

### PR DESCRIPTION
**Pull request by lihongyan1**
Automated 2 test cases
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-29061
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-29300
https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-21148
Dependent on PR:
https://github.com/openshift/cucushift/pull/7735
v3 job:
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/92058/console

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @yapei Please help review

File: lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml [hunk 1]
```diff
@@ -60,6 +76,17 @@ silence_alert_from_detail:
   action: input_silence_comments
   action: perform_silence
   action: check_silence_detail
+silence_alert_from_detail_check_fields:
+  params:
+    comment_text: silence comments is a must from ocp 4.50
```
Comment (juzhao) [inline lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml:81]: 4.50 -> 4.5

File: lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml [hunk 2]
```diff
@@ -60,6 +76,17 @@ silence_alert_from_detail:
...
+    duration: 6h
+  action: click_silence_alert
+  action: set_silence_duration
+  action: uncheck_start_from_now
+  action: check_regular_expression_link
```
Comment (juzhao) [inline lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml:86]: how about checked_regular_expression

File: lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml [hunk 3]
```diff
@@ -42,6 +45,19 @@ click_silence_alert:
...
+      xpath: //span[text()='Duration']/ancestor::div[@class='co-m-pane__body-group']//input[@type='checkbox']
+    op: click
+ensure_start_from_now_checked:
+  element:
+    selector:
+      xpath: //span[text()='Duration']/ancestor::div[@class='co-m-pane__body-group']//input[@type='checkbox'][@checked]
+check_regular_expression_link:
+  element:
+    selector:
+      xpath: //a[@href='https://github.com/google/re2/wiki/Syntax']
```
Comment (juzhao) [inline lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml:60]: why the xpath is that?

File: lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml (hunk 2 above)
Comment (lihongyan1) [reply lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml:86]: I suppose check is better, here what I want to do is check is the link exists

File: lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml (hunk 3 above)
Comment (lihongyan1) [reply lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml:60]: here I want to check the value of attribute href is what I want.

File: lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml (hunk 2 above)
Comment (juzhao) [reply lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml:86]: ok

**Pull request merged**